README.md

```diff
@@ -14,14 +14,13 @@ This project runs on Python 3.5+. To install required Python packages via `pip`,
 environment](#creating-a-python-virtual-environment); this ensures you do not install the packages globally. Note there
 are additional [requirements](./%7B%7B%20cookiecutter.repo_name%20%7D%7D/README.md#requirements) for the output.
 
-To create a new repository structure from this cookiecutter, install the required Python packages via `pip` by running
-the following command:
+To create a new repository structure from this cookiecutter, install the required Python packages via `pip` by running the following command in your terminal (macOS), shell (Linux) or console (Windows):
```

Comment:
We haven’t tested it on a Window machine. The requirements to use the template at the moment are to use a Unix-like operating system (macOS, Linux, …); see [here](https://github.com/ukgovdatascience/govcookiecutter/tree/feature/r-setup/%7B%7B%20cookiecutter.repo_name%20%7D%7D)

Unless you tested it?

I would play it safe and not mention Window unless we get a chance to test it on a Window machine.
```suggestion
To create a new repository structure from this cookiecutter, install the required Python packages via `pip` by running the following command in your terminal (macOS) or shell (Linux):
```